src/gui/widgets/Fader.cpp

```diff
@@ -60,11 +60,16 @@
 namespace lmms::gui
 {
 
+constexpr auto c_dBScalingExponent = 3.f;
+//! The dbFS amount after which we drop down to -inf dbFS
+constexpr auto c_faderMinDb = -120.f;
```

Comment:
[I would put these in an anonymous namespace.](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf22-use-an-unnamed-anonymous-namespace-for-all-internalnon-exported-entities)

Reply:
Done with commit 74d33a69b29.